rhv_checks.py

```diff
@@ -338,6 +340,55 @@ def check_vms_distributed_hosts(system, warn=5, crit=10):
         sys.exit(3)
 
 
+def check_hosted_engine_status(system, **kwargs):
+    """ Check the status of all the host's Hosted Engine Status."""
+    okay, critical, unknown, over_all = [], [], [], []
+    command = "hosted-engine --vm-status --json"
+    hosts = system.api.system_service().hosts_service().list()
+
+    for host in hosts:
+        output = ssh_command(host.name, password, command)
+        #Output is in the format of List of JSON removing newline.
+        if len(output) != 0:
+            result = output[0].replace("\\n", "").replace("\n", "")
+            host_data_dict = json.loads(result)
+            # Iterating value from Dict --> Dict --> Value.
+            for value in host_data_dict.values():
```

Comment:
I would change this to 
```python,
for host_index, host_detail in host_data_dict.items():
    # disregard the "global_maintenance" field
    if host_index == "global_maintenance":
        continue 
    name = host_detail["hostname"]
    status = host_detail["engine-status"]["health"]
     ### appends to list based on conditions
```
Maybe put this in a `try/except KeyError` block if those keys aren't valid.

Reply:
@john-dupuy `try/except` is not require as we can have `isinstance` checking with `dict`. If it is not a dict then will not encounter any `KeyError`, what's say .?

Reply:
`if` block still returning `TypeError`

Reply:
@sbulage which `if` block?

Reply:
Discussed offline with @john-dupuy.